### ISSUE TITLE
Allow deriving traits on the Error

### ISSUE THREAD

**Issue by Yamakaky** [brson/error-chain#134]
Possible syntax:
```
error_chain! {
  derive { Eq, PartialEq }
  ...
}
```

**Comment by TedDriggs** [comment]
Should this be done as "attrs" or something that would also allow passing attributes for proc macro crates like `serde_derive`?

**Comment by kpcyrd** [comment]
Any news? Ran into this problem while I was trying to use [pseudo](https://github.com/iredelmeier/pseudo) :)

**Comment by ivanovaleksey** [comment]
Hello, are there any updates in this?

**Comment by lneicelis** [comment]
Any updates?

**Comment by kpcyrd** [comment]
I've switched to failure in the meantime, and then switched to anyhow after that, I'm not sure if people still work on error-chain

**Comment by Yamakaky** [comment]
no, anyhow + thiserror is better so I don't think there is more work on error-chain.

**Comment by AndyGauge** [comment]
error-chain does not have new features developed, but there is an effort to keep it working as breaking changes are introduced from rust-lang.  Probably better to find a different library (if you even need one) for new projects.

**Comment by lneicelis** [comment]
Thank you!
